modules/utxo-lib/test/integration_local_rpc/generate/regtestNode.ts

```diff
@@ -79,6 +79,7 @@ export async function getRegtestNode(network: Network): Promise<Node> {
     dockerParams.image,
     ...(dockerParams.binary ? [dockerParams.binary] : []),
     '-regtest',
+    '-txindex',
```

Comment:
Why is `txindex` necessary with this PR?

Reply:
because I am looking up confirmed input transactions now